README.md

```diff
@@ -51,14 +51,72 @@ The only prerequisites for running it are [Nextflow](https://www.nextflow.io/) a
 
 <a name="install"></a>
 
-## Install and dependencies
+## Install and downloading dependencies
 
 To get a copy of the Mobilome Annotation Pipeline, clone this repo by:
 
 ```bash
 $ git clone https://github.com/EBI-Metagenomics/mobilome-annotation-pipeline.git
 ```
 
+The first time you run the pipeline you will need to set up the following databases:
+
+1. Download and index the database for amrfinder plus
+```bash
+wget -r -nH --cut-dirs=5 ftp://ftp.ncbi.nlm.nih.gov/pathogen/Antimicrobial_resistance/AMRFinderPlus/database/3.12/2024-01-31.1/
+singularity pull ncbi-amrfinderplus_3.12.8.sif docker://depot.galaxyproject.org/singularity/ncbi-amrfinderplus:3.12.8--h283d18e_0
+singularity exec -B $(pwd):/data ncbi-amrfinderplus_3.12.8.sif amrfinder_index 2024-01-31.1
```

Comment:
This instructions are specific for singularity, loads will use docker or conda to install this. I think these instructions are better suited -> https://nf-co.re/funcscan/2.1.0/docs/usage/#amrfinderplus